### PR TITLE
adding Matamo Analytics to Across

### PR DESCRIPTION
**Pull request by Tommy1231232**
This is for tracking page views on Across from Matamo, an analytics tool. https://matomo.org/

FYI directions on integrating Matamo:
Screen Shot 2022-02-01 at 2 57 58 PM

### CODE REVIEW

**Comment by Tommy1231232** [comment]
Note - this is only for tracking page views.

**Review by daywiss — CHANGES_REQUESTED** [review]
this actually should just be included on the index.html page. this app is considered a "SPA" (single page application), so these files you have edited only represent pages conceptually. The whole app is actually rendered within index.html, linked here https://github.com/across-protocol/across-frontend/blob/AcrossAnalytics/public/index.html

**Comment by Tommy1231232** [comment]
> this actually should just be included on the index.html page. this app is considered a "SPA" (single page application), so these files you have edited only represent pages conceptually. The whole app is actually rendered within index.html, linked here https://github.com/across-protocol/across-frontend/blob/AcrossAnalytics/public/index.html

Thanks! I updated in another commit.

File: public/index.html [hunk 1]
```diff
@@ -1,6 +1,23 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
+
+<!-- Matomo -->
+<script>
```
Comment (daywiss) [inline public/index.html:5]: small nit, in the directions they say to paste this right before the closing head tag, that looks like `</head>`, so i think they want it right above this line: https://github.com/across-protocol/across-frontend/pull/150/files#diff-c2cc24bc9001b11b6add48a4cd8f893d5d6c6e4d1bd254158bd14ab997f552cdR56

File: src/views/About.tsx [hunk 2]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (daywiss) [inline src/views/About.tsx:1]: these mean an extra line was introduced, you can just delete the line

File: src/views/Pool.tsx [hunk 3]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (daywiss) [inline src/views/Pool.tsx:1]: same here

File: public/index.html (hunk 1 above)
Comment (Tommy1231232) [reply public/index.html]: Thank you! Done

File: src/views/About.tsx (hunk 2 above)
Comment (Tommy1231232) [reply src/views/About.tsx]: Removed line

File: src/views/Pool.tsx (hunk 3 above)
Comment (Tommy1231232) [reply src/views/Pool.tsx]: Removed line

File: public/index.html [hunk 4]
```diff
@@ -36,6 +36,21 @@
 
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo-small.png" />
     <title>Across - Bridge Layer 1 and Layer 2 assets</title>
+    <!-- Matomo -->
+<script>
```
Comment (daywiss) [inline public/index.html:40]: this script block (everything including `<script>...</script>` and inside) needs to be tabbed to line up with the `<title>` block above, just to keep things a bit neat. but outside of that looks ready to go

Comment (Tommy1231232) [reply public/index.html]: Updated. Thank you!

**Review by daywiss — APPROVED** [review]

**Pull request merged**